### ISSUE TITLE
Use schema description to populate class docstring

### ISSUE THREAD

**Issue by Liam-Deacon** [koxudaxi/datamodel-code-generator#274]
I'd like to be able to retain my openapi schema descriptions in the generated python code, e.g.

```yaml
components:
  schemas:
    MyClass:
      description: >-
        My classes description goes here...
      type: object
      properties: {}
```

Which I would like to be preserved in the conversion process to python, e.g.

```python
class MyClass(BaseModel):
    """
    My class description goes here...
    """
    ...
```

Is this easy to implement? I'd be happy to raise a PR, I'm not sure where to start looking in the codebase though!

**Comment by koxudaxi** [comment]
@Liam-Deacon 
Thank you for posting this issue.
It's a good feature.

I explain where do we must change.

We should change the jinja2 template to write a description of the classes.
https://github.com/koxudaxi/datamodel-code-generator/blob/43bdc6239126769d27f1ecd54a624003f970d31c/datamodel_code_generator/model/template/pydantic/BaseModel.jinja2#L4-L5

Also, `DataModel` needs the `description` to give it to the jinja2 template
https://github.com/koxudaxi/datamodel-code-generator/blob/43bdc6239126769d27f1ecd54a624003f970d31c/datamodel_code_generator/model/base.py#L115-L121

And, We must set the `description` of the classes on data_model contractor.
https://github.com/koxudaxi/datamodel-code-generator/blob/43bdc6239126769d27f1ecd54a624003f970d31c/datamodel_code_generator/parser/jsonschema.py#L497-L499

If I have the time then, I will implement it. 
But, I'm busy to try other requests :( 
PR welcome!!

**Comment by koxudaxi** [comment]
@Liam-Deacon 
I have released a new version as `0.6.15`
I added `--use-schema-description` option on it for realizing your idea.
Thank you.
